src/index.css

```diff
@@ -0,0 +1,18 @@
+table , td, th {
```

Comment:
It seems that these styles are not used anywhere. Can you eliminate this?

Reply:
These styles are used by index.js for displaying block list.
So it cannot be deleted.